### PR TITLE
GCE: pass GCE_ALPHA_FEATURES environment variable

### LINKED ISSUE #51976: gce-alpha-api test suite failing
The environment variable GCE_ALPHA_FEATURES is not set correctly in the test custer.

### PR DESCRIPTION
**Pull request by yujuhong**
This fixes #51976

This PR fixes the tests in https://k8s-testgrid.appspot.com/sig-network#gce-alpha-api

### CODE REVIEW

**Comment by yujuhong** [comment]
/cc 1.8 branch manager @abgworrall 
This is a bug fix. Is it ok to merge this?

File: cluster/common.sh [hunk 1]
```diff
@@ -678,6 +678,7 @@ PROMETHEUS_TO_SD_ENDPOINT: $(yaml-quote ${PROMETHEUS_TO_SD_ENDPOINT:-})
 PROMETHEUS_TO_SD_PREFIX: $(yaml-quote ${PROMETHEUS_TO_SD_PREFIX:-})
 ENABLE_PROMETHEUS_TO_SD: $(yaml-quote ${ENABLE_PROMETHEUS_TO_SD:-false})
 ENABLE_POD_PRIORITY: $(yaml-quote ${ENABLE_POD_PRIORITY:-})
+GCE_ALPHA_FEATURES: $(yaml-quote ${GCE_ALPHA_FEATURES:-})
```
Comment (bowei) [inline cluster/common.sh]: Take a look at PROVIDER_VARS, as we don't want to have cloud provider specific vars leak in to common.sh.

Comment (yujuhong) [reply cluster/common.sh]: Good point. Didn't know we had PROVIDER_VARS. Updated the PR

**Comment by yujuhong** [comment]
Updated the PR description with the issue.
/approve

**Pull request merged**